gammacat/input.py

```diff
@@ -355,11 +398,14 @@ def read(cls):
         schemas = Schemas.read()
         sources = BasicSourceList.read()
         datasets = InputDatasetCollection.read()
+        ref_infos = InputInfoCollection.read()
```

Comment:
`InputInfoCollection.read()` to have the collection of the info files as a variable of `InputDataCollection`. 
This is not being used but to have it similar to the other data files I added this line